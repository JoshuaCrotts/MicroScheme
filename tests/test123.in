;
; Convert a list of characters to a string.
;

(define x '(#\A #\p #\p #\l #\e))
x
(define y (list->string x))
(string-length y)
