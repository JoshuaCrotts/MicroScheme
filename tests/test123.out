(A p p l e)
5
